src/style/_models.scss

```diff
@@ -186,6 +187,13 @@ section.models
         {
             margin: 0 20px;
         }
+
+        .models-jump-to-path {
+          position: absolute;
```

Comment:
absolute? Is that safe ... it's been forever since I've dabbled in css / scss

Reply:
Yes, under a `relative` it is fine